UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/RNWService.cs

```diff
@@ -110,5 +110,24 @@ public async Task<List<WarningType>> GetWarningTypes()
         {
             return await _rnwRepository.GetWarningTypes();
         }
+
+        public async Task<List<RadioNavigationalWarningsData>> GetRadioNavigationalWarningsData(string correlationId)
```

Comment:
There are no unit tests covering this method 😞

Reply:
done required changes